### ISSUE TITLE
AttributeError: 'ChamberParm' object has no attribute 'LJ_14_radius'

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#746]
getting this when I tried to slice `ala_ala_ala.parm7`

travis log
```bash
  File "/home/travis/build/ParmEd/ParmEd/test/test_parmed_structure_slicing.py", line 274, in test_structure_box_and_space_group_and_symmetry
    assert_correctly_copy(parm)
  File "/home/travis/build/ParmEd/ParmEd/test/test_parmed_structure_slicing.py", line 246, in assert_correctly_copy
    sliced_parm = parm['@1-3']
  File "/home/travis/miniconda/envs/myenv/lib/python2.7/site-packages/ParmEd-2.5.1+90.g1a64b1b-py2.7-linux-x86_64.egg/parmed/amber/_amberparm.py", line 413, in __getitem__
    other._set_nonbonded_tables()
  File "/home/travis/miniconda/envs/myenv/lib/python2.7/site-packages/ParmEd-2.5.1+90.g1a64b1b-py2.7-linux-x86_64.egg/parmed/amber/_chamberparm.py", line 654, in _set_nonbonded_tables
    self.recalculate_LJ()
  File "/home/travis/miniconda/envs/myenv/lib/python2.7/site-packages/ParmEd-2.5.1+90.g1a64b1b-py2.7-linux-x86_64.egg/parmed/amber/_chamberparm.py", line 310, in recalculate_LJ
    rij = self.LJ_14_radius[i] + self.LJ_14_radius[j]
AttributeError: 'ChamberParm' object has no attribute 'LJ_14_radius'
```

**Comment by swails** [comment]
I think I can fix this quickly

**Comment by swails** [comment]
Fixed in a PR I submitted to your branch (which is itself a PR here)

**Comment by swails** [comment]
Thanks for the report!

**Comment by swails** [comment]
Fixed in #744
